src/test/components/MarkerTable.test.js

```diff
@@ -62,21 +66,53 @@ describe('MarkerTable', function() {
     const store = storeWithProfile(profile);
     const renderResult = render(
       <Provider store={store}>
-        <MarkerTable />
+        <>
+          <MarkerContextMenu />
+          <MarkerTable />
+        </>
       </Provider>
     );
-    const { container } = renderResult;
+    const { container, getByText } = renderResult;
 
     const fixedRows = () =>
       Array.from(container.querySelectorAll('.treeViewRowFixedColumns'));
     const scrolledRows = () =>
       Array.from(container.querySelectorAll('.treeViewRowScrolledColumns'));
 
+    const node = str => getByText(str);
+    const row = str =>
+      ensureExists(
+        node(str).closest('.treeViewRow'),
+        `Couldn't find the row for node ${String(str)}.`
+      );
+    const contextMenu = () =>
+      ensureExists(
+        container.querySelector('.react-contextmenu'),
+        `Couldn't find the context menu.`
+      );
+
+    const click = (element: HTMLElement) => {
```

Comment:
What's going on here with the multiple events?

Reply:
I believe it's because of the multiple events being listened to from the context menu component? There should probably be some code comments explaining it.